Sources/FoundationEssentials/Predicate/Expressions/Sequence.swift

```diff
@@ -123,6 +123,7 @@ extension PredicateExpressions {
         }
     }
     
+    @_disfavoredOverload
```

Comment:
This is used to disambiguate between `Sequence.contains` and `RangeExpression.contains` - in this case, we prefer the more specific implementation provided by `RangeExpression`, just as the compiler would if you were calling `.contains` on an instance